sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -380,6 +404,20 @@ public void sendRPCs(List<? extends RPCMessage> rpcs, final OnMultipleRequestLis
 
 	// LIFECYCLE / OTHER
 
+	// STARTUP
+	public void start(CompletionListener listener){
```

Comment:
Please add javadoc to this method to inform the developer of what they can expect by calling it.